src/components/services/NotificationService.ts

```diff
@@ -61,23 +60,26 @@ export const updateNotificationInfo = async (
  */
 export const NotificationService = (props: Props) => {
   const { account } = props
+
   const { notifState, accountNotifDismissInfo } = useAccountSettings()
+  const { lastChecked, dontAsk } = useOtpSettings()
```

Comment:
The variable names are pretty generic, and this component deals with a lot of unrelated stuff. We should make it clear that they pertain to OTP, specifically. I would either do:

```ts
const optSettings = useOtpSettings()

... optSettings.dontAsk ...
```

Or I would do:

```ts
const { lastChecked: otpLastChecked, dontAsk: otpDontAsk } = useOtpSettings()
```

Either way is fine.